ui/app/styles/components/selectable-card.scss

```diff
@@ -67,6 +67,11 @@
     font-weight: 500;
     line-height: 1.33;
   }
+
+  .vlt-table {
+    max-height: 240px;
+    overflow: scroll;
```

Comment:
A couple thoughts.  One to prevent the scroll bar from showing on the x axis, you might want to add either overflow-y: scroll instead of overflow scroll, or add overflow-x: hidden.

I worked on this one a little and got it so the header was sticky and scrolled.  It makes it a little harder, but I'd be happy to pair quickly through the solution if you think it's worth it.  The header being : 
![image](https://user-images.githubusercontent.com/6618863/83890926-abf92e80-a709-11ea-9027-bfb3d2ee2dec.png)